scripts/run.py

```diff
@@ -149,6 +150,8 @@ def main(backend, path, url, model_point):
       https://github.com/pyhf/pyhf-benchmark
 
     """
+    if backend == "tensorflow":
+        tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
```

Comment:
`pyhf` requires TensorFlow `v2` so there's no need to use the TensorFlow v1.0 compatible API.

Reply:
Updated